Userland/Libraries/LibGUI/WindowServerConnection.h

```diff
@@ -58,6 +58,7 @@ class WindowServerConnection final
     virtual void track_mouse_move(Gfx::IntPoint const&) override;
     virtual void ping() override;
 
+    bool m_in_command_pallette { false };
```

Comment:
Minor spelling error: "palette", not "pallette". Same with the commit message.

Reply:
Fixed.

Reply:
Commit message still says "pallette"

Reply:
Should be OK now.